packages/realm-server/scripts/start-base-root.sh

```diff
@@ -9,6 +9,10 @@ NODE_ENV=development \
   PGPORT=5435 \
   PGDATABASE=boxel_test_base_root \
   REALM_SECRET_SEED="shhh! it's a secret" \
+  REALM_SERVER_SECRET_SEED="shhh! it's a secret" \
```

Comment:
like i mentioned above, i'm not totally convinced having 2 seeds makes things more secure, and probably just extra hassle